forge/src/main/java/org/openjdk/skara/forge/PullRequest.java

```diff
@@ -181,8 +181,8 @@ default boolean isSame(PullRequest other) {
     }
 
     /**
-     * Get the last force-push time. If there is no force-push in pull request
-     * or the restful api doesn't support force-push, return empty.
+     * Return the last time something was force pushed while not in draft state.
+     * If there is no force-push in pull request or the restful api doesn't support force-push, return empty.
```

Comment:
Try to keep javadoc comments to 80 char line width.

Reply:
OK, will fix it!